src/python/WMCore/ReqMgr/DataStructs/DefaultConfig/EDITABLE_SPLITTING_PARAM_CONFIG.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function, division
+#TODO: need to get algorithm proper way
+ALGO_DICT = {"FileBased": {"files_per_job": ''},
+             "TwoFileBased": {"two_files_per_job": ''},
+             "LumiBased": {"lumis_per_job": '', 
+                          "halt_job_on_file_boundaries": True},
+             "EventAwareLumiBased": {"avg_events_per_job": '',
+                                     "max_events_per_lumi": '',
+                                     "halt_job_on_file_boundaries": True},
+             "EventBased": {"events_per_job": '',
+                            "events_per_lumi": '',
+                            "lheInputFiles": False},
+             "Harvest": {"periodic_harvest_interval": ''},
+             "ParentlessMergeBySize": {"min_merge_size": '',
+                                       "max_merge_size": '',
+                                       "max_merge_events": '',
+                                       "max_wait_time": ''},
+             "WMBSMergeBySize": {"min_merge_size": '',
+                                 "max_merge_size": '',
+                                 "max_merge_events": '',
+                                 "max_wait_time": ''},
+             "MergeBySize": {"min_merge_size": '',
+                             "max_merge_size": '',
+                             "max_merge_events": '',
+                             "max_wait_time": ''}
+             }
+
+ALGO_LIST_BY_TYPES = {"Processing": ["LumiBased", "EventAwareLumiBased", 
```

Comment:
Not being picky..., but can you reformat (spacing) this block. PyCharm works like a charm for this case ;)